Geometry/service/Geometry.fcl

```diff
@@ -2,7 +2,7 @@ BEGIN_PROLOG
 
 standard_geometry:
 {
-  GeoFileName: "gdml/phase1b.gdml"
+  GeoFileName: "gdml/phase1c.gdml"
```

Comment:
This actually doesn't get used anywhere - I searched for GeoFileName and this is the only file it's in. We use RunHistory to determine which geometry we need to use.

Reply:
changed it back, since it doesn't make a difference